test/unittest.sh

```diff
@@ -18,11 +18,10 @@ export NNI_PLATFORM=unittest
 echo ""
 echo "===========================Testing: nni_sdk==========================="
 cd ${CWD}/../src/sdk/pynni/
+python3 -m pip install --user -r nni/smac_tuner/requirements.txt
 python3 -m unittest discover -v tests
 
```

Comment:
This unittest.sh is used by window IT pipeline,  can smac requirements be installed on Windows?

Reply:
SMAC cannot be used on Windows, as specified in documentation. Suggestions?

Reply:
You can install the packages in pipeline instead of this unittest.sh.   exclude the smac unit test cases on windows.

Reply:
It seems SMAC is already installed in pipelines by
`
      PATH=$HOME/.local/bin:$PATH nnictl package install --name=SMAC
`